workflows/proteomics/clinicalmp/clinicalmp-quantitation/README.md

```diff
@@ -0,0 +1,20 @@
+# Clinical Metaproteomics 4: Quantitation
+
```

Comment:
Could you say a little bit more about the workflow, when you would run it, and what the number 4 refers to ? If you have a link to a GTN tutorial that would also be something you can add here.